docs/config.rst

```diff
@@ -41,3 +41,19 @@ An environment variable like ``REACT_APP_FOO`` controls the setting ``FOO``.
     :default: ``https://${OIDC_DOMAIN}/userinfo``
 
    The audience for the access token that is generated by Auth0.
+
+
+.. envvar:: REACT_APP_REFRESH_AUTH_SESSION_SECONDS
+
+    :default: 300
+
+    How often to issue a silent authentication refresh.
+
+.. envvar:: REACT_APP_REFRESH_PREEMPTIVE_SECONDS
+
+    :default: 900
+
+    How soon *before* the session expires do we perform a silent authentication to keep it fresh.
+    Setting this liberally means we can make sure the access token is up-to-date *before* the
```

Comment:
is this a typo, and should it read `literally` instead of `liberally`? Not sure...

Reply:
I meant liberally. But now I see that that's a poor choice of words. The intention was to say "settings this QUITE HIGH means..." or something like that. I'll think of a better way of phrasing it.